index.html

```diff
@@ -108,9 +109,168 @@ <h5 class="card-heading">Web Development</h5>
         </div>
 
 
+    </section>
+    <section class="video-section">
+        <div class="container__video">
+            <div class="container heading">
+                <h2 class="heading--two">
+                    We are providing best business service
+                </h2>
+                <p class="choose-us__heading--p">Problems trying to resolve the conflict between</p>
+                <p class="choose-us__heading--paragr"> the two major realms of Classical physics: Newtonian mechanics</p>
+            </div>
+            <div class="container__video--player">
+                <video  controls>
+                    <source  src="video/work_space_-_3132%20(720p).mp4" type="video/mp4">
+                    Your browser does not support HTML video.
+                  </video>
+            </div>
+        </div>
+
+    </section>
+    <section class="client-say">
+        <div class="container__client-say">
+            <div class="heading">
+                <h2 class="heading--two">
+                    What client say
+                </h2>
+                <p class="choose-us__heading--p">
```

Comment:
good for using same class. But as you can see now the name of the class doesn't fit the parent anymore. As it is not choose-us section. So my suggestion to find a more generic class name that will fit any section